test/dummy/config/database.yml

```diff
@@ -28,8 +28,6 @@ default: &default
   adapter: mysql2
   username: root
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  host: "127.0.0.1"
-  port: 33060
```

Comment:
This won't work with the `mysql` service in `docker-compose.yml`: 
https://github.com/basecamp/solid_queue/blob/2a5256842b974c276e3dc0e8e83abb012351e37e/docker-compose.yml#L7-L13

Reply:
using docker-compose sudenlly makes things work much better :D I still think the default mysql port should be used but docker-compose works

Reply:
If you run `bin/setup`, it'll run `docker-compose` for you: 
```bash
docker-compose up -d --remove-orphans
docker-compose ps
```

The different port choice is so as not to interfere with any local MySQL server you might be running in the host. The container port is the default.